<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCcontent_twoMemories_Form</class>
 <widget class="QWidget" name="DCcontent_twoMemories_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Diagnostic Codes</string>
  </property>
  <layout class="QVBoxLayout" name="main_verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="tables_verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="currOrTempDTCsTitle_label">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Current/Temporary Diagnostic Trouble Code(s):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="currOrTempDTCs_tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>Code:</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description:</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="histOrMemDTCsTitle_label">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Historic/Memorized Diagnostic Trouble Code(s):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="histOrMemDTCs_tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>Code:</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description:</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="DCclearingInfo_label">
       <property name="text">
        <string>=&gt; Call ''Clear Memory'' from the ''Selection'' menu to erease all DTCs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printDClist_pushButton">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>    Print    </string>
       </property>
       <property name="icon">
        <iconset resource="../resources/FreeSSM.qrc">
         <normaloff>:/icons/chrystal/32x32/fileprint.png</normaloff>:/icons/chrystal/32x32/fileprint.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/FreeSSM.qrc"/>
 </resources>
 <connections/>
</ui>
